dev/breeze/src/airflow_breeze/commands/testing_commands.py

```diff
@@ -1010,6 +1017,13 @@ def system_tests(
     python: str,
     skip_docker_compose_down: bool,
     test_timeout: int,
+    use_airflow_version: str,
+    airflow_constraints_reference: str,
+    clean_airflow_installation: bool,
+    force_lowest_dependencies: bool,
+    install_airflow_with_constraints: bool,
+    distribution_format: str,
+    use_distributions_from_dist: bool,
```

Comment:
Why adding all those parameters, you seem to use only `use_airflow_version`?

Reply:
It makes sense to cover existing functionality - but yes - all those params should be passed-through to `ShellParams` as they might be used for various cases - for example we could build CI image for tests in `v3-0-test` -there we just pull packages from PyPI rather than build CI images locally etc. etc.